rom/dev/tests/rom_integration_tests/test_fmcalias_derivation.rs

```diff
@@ -331,14 +333,14 @@ fn test_pcr_log_fmc_fuse_svn() {
         .owner_pubkey_digest(&image_bundle.manifest.preamble)
         .unwrap();
 
-    const FMC_SVN: u32 = 3;
-    const FMC_FUSE_SVN: u32 = 2;
+    const FW_SVN: u32 = 3;
+    const FW_FUSE_SVN: u32 = 2;
 
     let fuses = Fuses {
         anti_rollback_disable: false,
         key_manifest_pk_hash: vendor_pubkey_digest,
         owner_pk_hash: owner_pubkey_digest,
-        fmc_key_manifest_svn: FMC_FUSE_SVN,
+        fw_svn: [2, 0, 0, 0], // Value of FW_FUSE_SVN
```

Comment:
nit: Would be nice to have a function do this instead of keeping them in sync manually

Reply:
Have noted as a TODO.